thesis/inc/introduction/game_industry.tex

```diff
@@ -0,0 +1,34 @@
+\section{Game Industry}
+As developers of quality interactive experiences, modern video game developers face a fairly unique
+combination of challenges.
+
+\subsection{Flexibility}
+A valued concept within the video game industry is fast iteration cycles.
+Game developers often want to be able to test out and iterate on their ideas quickly.
+Fast iteration cycles require a great deal of flexibility,
+it is therefore desirable to keep software modules decoupled and generic,
+allowing developers to quickly combine different modules into new combinations creating new functionality,
+and content.
+
+\subsection{Re-usability}
```

Comment:
No need for the - between re and usability. Reusability is a perfectly valid word by itself